ParquetClassLibrary/Sandbox/MapRegionEditor.cs

```diff
@@ -224,25 +225,26 @@ public bool PaintAtLocation(Vector2Int in_position)
 
             if (_parquetPaintPattern.IsSet(ParquetMask.Floor))
             {
-                error += _currentRegion.TrySetFloor(new Floor(_floorToPaint), in_position)
+                // TODO: Should I be using .MemberwiseClone() here to duplicate the parquet?
```

Comment:
Structs are pass by value, so there's no need to use MemberwiseClone for ParquetID. If you had a reference value inside the Struct, then maybe. Quote Panda: Structs probably shouldn't be including complicated, non-serializeable values like reference values.

Reply:
Thanks, good catch.  Fixed as of gffe51d.